types/serverless-step-functions/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for serverless-step-functions 3.14
+// Project: https://github.com/serverless-operations/serverless-step-functions
+// Definitions by: Chris Cook <https://github.com/zirkelc>
+//                 Enric Bisbe Gil <https://github.com/ebisbe>
```

Comment:
this requires explicit ACK from this person, as owners are notified and part of automated communication from DT repo. You can give a credit/kudos instead, when lacking ack

Reply:
These are my name and @ebisbe - I could add @massfords as well if he's interested?

Reply:
@peterblazejewicz I'm ok